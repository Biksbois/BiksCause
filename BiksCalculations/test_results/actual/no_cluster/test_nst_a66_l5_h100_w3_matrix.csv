,Clouds,Clear,sky is clear,overcast clouds,broken clouds,few clouds,scattered clouds,traffic_volume_2,traffic_volume_0,traffic_volume_1
Clouds,0.99,0.89,0.89,0.82,0.73,0.83,0.78,1.14,0.65,0.76
Clear,1.04,1.0,1.0,1.25,2.04,0.71,0.38,0.87,1.07,0.94
sky is clear,1.04,1.0,1.0,1.25,2.04,0.71,0.38,0.87,1.07,0.94
overcast clouds,1.25,0.65,0.65,1.06,0.94,,1.22,1.36,1.03,0.99
broken clouds,1.21,0.6,0.6,0.86,0.94,1.34,1.12,1.25,0.77,1.17
few clouds,1.32,1.44,1.44,,0.75,1.0,0.68,1.86,0.57,0.64
scattered clouds,1.2,1.85,1.85,0.82,0.77,2.07,1.0,1.36,0.84,1.28
traffic_volume_2,0.87,1.08,1.08,0.81,0.85,0.52,0.74,0.99,,0.77
traffic_volume_0,1.36,0.9,0.9,0.97,0.98,1.0,1.19,,1.0,0.96
traffic_volume_1,1.42,1.08,1.08,1.13,0.67,1.65,0.62,1.92,0.67,1.0
