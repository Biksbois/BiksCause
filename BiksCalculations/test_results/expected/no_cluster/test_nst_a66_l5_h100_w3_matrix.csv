,Clouds,Clear,traffic_volume_2,traffic_volume_0,traffic_volume_1,sky is clear,overcast clouds,broken clouds,few clouds,scattered clouds
Clouds,0.99,0.89,1.14,0.65,0.76,0.89,0.82,0.73,0.83,0.78
Clear,1.04,1.0,0.87,1.07,0.94,1.0,1.25,2.04,0.71,0.38
sky is clear,1.04,1.0,0.87,1.07,0.94,1.0,1.25,2.04,0.71,0.38
overcast clouds,1.25,0.65,1.36,1.03,0.99,0.65,1.06,0.94,,1.22
broken clouds,1.21,0.6,1.25,0.77,1.17,0.6,0.86,0.94,1.34,1.12
few clouds,1.32,1.44,1.86,0.57,0.64,1.44,,0.75,1.0,0.68
scattered clouds,1.2,1.85,1.36,0.84,1.28,1.85,0.82,0.77,2.07,1.0
traffic_volume_2,0.87,1.08,0.99,,0.77,1.08,0.81,0.85,0.52,0.74
traffic_volume_0,1.36,0.9,,1.0,0.96,0.9,0.97,0.98,1.0,1.19
traffic_volume_1,1.42,1.08,1.92,0.67,1.0,1.08,1.13,0.67,1.65,0.62
